src/commands/releases.rs

```diff
@@ -516,67 +515,44 @@ fn execute_set_commits<'a>(
         }
         ctx.api.set_release_refs(&org, version, heads)?;
     } else {
-        println!("No commits found. Leaving release alone.");
-    }
+        println!("Could not determine any commits to be associated with a repo-based integration. Proceeding to find commits from local git tree.");
```

Comment:
This is also a feature that should be enabled for `--local` only. Right now, when someone uses `--auto` and head will be missing, it'll execute this code path as well.